### PR TITLE
fix xss detect mistake for on event

### PR DESCRIPTION
**Pull request by zoulinz**

### CODE REVIEW

File: xss_parser.go [hunk 1]
```diff
@@ -321,9 +351,10 @@ func is_black_attr(s string, len int) int {
 
 	if len >= 5 {
```
Comment (donge) [inline xss_parser.go:352]: 具体误伤的例子能不能描述一下

Comment (zoulinz) [reply xss_parser.go:352]: "X-S3-Sessionid": [
			"TS0147f4c6=0152b77e2817ad0631f20e4e698edfe4f48e383576b09b75719067f4378b2dd2672a447b03bdd0cc4607e5b87c916c3ed06bfdfa04; TS019d85d5=0152b77e2817ad0631f20e4e698edfe4f48e383576b09b75719067f4378b2dd2672a447b03bdd0cc4607e5b87c916c3ed06bfdfa04; _icla=4060312647971782022.991371482; TS01f48049=0152b77e281525cd665ee78e5835cb03e9c39521414d7b70cd1bbb9eda2918814337163d53607cd31bf71c15d6e8826fcb4d699d6a; c__utma=4060312647971782022.991371482.9185.1662806107.1662806107.1; c__utmb=4060312647971782022.991371482.1662806107.1662806107.1; c__utmc=4060312647971782022.991371482; s_u_id=; x-s3-sid=S1n9EqkjhmH0t14r21cpaktzw; x-s3-rtk=Tx21cpaktu9HSkjhmVLt1GvYi01r9RacNry4zk; x-s3-tid=b91091517a034ad81db7b0851a2a10cff838ab13:GvYi01_cNry:3d546dbe-30f4-11ed-aa22-d96389f3500f:ae97f56c0c; oneapmclientid=18326f7aa6d62f-01e2048c46eaed-484f7910-51a6b-18326f7aa6e747; x-s3-s4e=lxwnoSKuF48lVHNW0XGc1QOsBJd1RCb02UhdPWJxXYGjQvfE1DhrgeftzOb1NxPFHYV0TKKl7TB%2BWF5MlecDZc%2BmKZMuZJTqkKiOmQ5ttbSJIZ6Aq6eHUxomTHfY9aWydOFjjgsoEEC49XW5qTW7tYTdkcq%2Bzu2T4xDwCqWwyLOKGv9lHFSNbsnXP84cNAHDaF8xBnDB06jG4ZL9YyF%2FcASJSqO8VmmV2CkGhfSKWOxKsBomZ8Sj6lXLhlyV1OogtC%2F4Sk%2FqYe1RyqU9V0qPLn25pF%2BHFNAbijuWLMhZFMaVbFXgjlcIOq1ipI%2BnAhDasbGjAUPlLij7Ch%2F3csMnSumFzm6oJArUcp6LVaxdhPja0HTevbNRoSDJYbGOsX6H7Rq%2FqijjVe%2BCaqvb1jOVt7YK9Itepwh8N1uWfQza17PiM9mw77X%2FsEd2nwKENpcIcGvKrzYsw553zwDWVkZrvy97SV5q3LrgKHlpui8CY9K4YnsqQpKMBM8nwOSud2ttz2d3zUA50eExk7lgjJC0z6GLIDcl5YPX%2Bn2Jtn%2Bk8cXpF7c6TGg91g%2B%2Fl8GMJyBxahHT0pM5m8E2VeJtdBdxwp2gVTtgsr%2BjSwE1Yb0%2BWleBTE2vHp%2B18IlYOG1hL1k6pPNHFldKRGxmy2V6kShE6DDIaQd8hbBEO7vWWojWmsN3IlmxJpz1kEs0CW6M9B2lpacCX%2Fv1AlI1E7ojmfZSWkLjOz988wXoFMUEXcLIc9mKeAoShgp1WveG3q%2BcE1NEm0qbf0nNHvCleUw4K7wSU%2Bm3Ql9bliIB675QsLfoAIa9gm98DUi4zE%2FscmRKEOy2wkW44UkKzZDftj%2BFbmb9%2FUjZO4ro5boFIusIjY8FUfscZIurN2CVVEnv0PqipybnA6bJogW1HZ%2BjMVaLshN4CpMr%2BO35q8KY%2BOOzx%2BQQJsg1CNdFOiJROj234hQ4AgzL3uRGU6HV1ScbpaSBX0lToo4iphe6zsu%2B30JjqhvHGLjygj4sxaMDjMU2pnKMoRvSWThc89%2FRuqyXnZ12h7jlgqFh5ilmMvH27s6Inf43ncQF1VCT1gJxNU8Y3VKyCnm1lPslmzg8Nc3o0Q8b4USUFq0qtXkmhf0WIaUzWIUXttj%2BNR8gQ0OM4g8PPVBS1S4E3sSsb91091517a034ad81db7b0851a2a10cff838ab13:GvYi01_cNry:3d546dbe-30f4-11ed-aa22-d96389f3500f:ae97f56c0c"
		],
误伤里面检测到有一个属性 oneapmclientid= 触发了on event 检测。 认为是类似onerror=alert()

**Comment by zoulinz** [comment]
fixed

**Review by donge — APPROVED** [review]

**Comment by donge** [comment]
Nice shot

**Pull request merged**